src/minuteman_metrics.erl

```diff
@@ -377,8 +375,8 @@ process_p99s_4_test_() ->
 process_dst_ip_map_test_() ->
     DAddr = {54, 192, 147, 29},
     IP = 16#36c0931d,
-    Conn = {ip_vs_conn, tcp, established, 167792566, 47808, IP, 8080, 167792566, 8081, 59},
-    Conn2 = {ip_vs_conn, tcp, established, 167792567, 47808, IP, 8080, 167792566, 8081, 59},
+    Conn = {ip_vs_conn, tcp, established, 167792566, 47808, 167792566, 8080, IP, 8081, 59},
```

Comment:
Yeah, I'm pretty sure that this representation is correct, I was just thinking that since the type is actually a record we should treat it as such. Not a big deal, I'm fine with merging as is.